doc/ref_impl/cntt-ri/chapters/chapter05.md

```diff
@@ -5,15 +5,248 @@
+ 
+<a name="5.3.1"></a>
+### 5.3.1 Hardware resource information
+The support of different workload types, each with different compute, storage, network requirements which needs kinds of hardware configuration template, we name this as server template.
+Besides it may include optional information such area name, data center name etc.
+
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| area_name | String | Yes | data center area name |
+| area_center_name | String | Yes | data center name, may compliance with a naming rule.  |
+| room_name | String | Yes | rome static data, helpful in locating issue occurred. |
+| city | String | Yes |  |
+| resource_pool_name | String | Yes |  |
+| outband_network_segment | String | Yes |  |
+| server_templates | List | Yes |  |
+| server_infos | List | Yes |  |
+
+<p align="center"><b>Table 5-3-1:</b> Hardware resource description.</p>
+
+### 5.3.2 Server template
+The server template describes the capability of the host like processor, memory, harddrive, raid, NIC/NIC binding, manufacturer, model etc.
+Server template would be assigned to multiple servers, i.e physical hosts.
+
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| template_name | String | Yes |  |
+| manufacturer | String | Yes |  |
+| model | String | Yes |  |
+| processor | String | Yes |  |
+| memory | String | Yes |  |
+| hard_drive | String | Yes |  |
+| raid | String | Yes |  |
+| network_controller_infos | List | Yes |  |
+| network_card_bond_infos | List | No | NIC bonding might not be always the case.|
+
+<p align="center"><b>Table 5-3-2:</b> Server template.</p>
+
+### 5.3.3 Server information
+Server will reference a server template, i.e, inherting all configuration of server template,
+Besides it may include additional information pim username, password, rack_name,position etc.
+
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| template_name | String | Yes | template referenced |
+| device_name | String | Yes | e.g NFV-D-HDBNJ-02A-3503-G-02-M-SRV-01 |
+| outband_ip | String | Yes |  |
+| outband_subnet_mask | String | Yes |  |
+| pim_username | String | Yes |  |
+| pim_password | String | Yes |  |
+| rack_name | String | Yes |  |
+| position | String | Yes |  |
+| remote_manange_ip | String | Yes | remote management ip, e.g. iLO,iDRAC, BMC |
```

Comment:
Typo: remote_manange_ip -> remote_manage_ip

Reply:
fixed in a new commit.

> Typo: remote_manange_ip -> remote_manage_ip

fixed in new commit